java/java.lsp.server/nbcode/integration/nbproject/project.xml

```diff
@@ -59,6 +59,21 @@
                         <specification-version>1.6</specification-version>
                     </run-dependency>
                 </dependency>
+                <dependency>
+                    <code-name-base>org.netbeans.modules.java.source.base</code-name-base>
+                    <build-prerequisite/>
+                    <compile-dependency/>
+                    <run-dependency>
+                        <implementation-version/>
```

Comment:
Ugly. Can we avoid implementation dependency in `nbcode`? Can `java.lsp.server` expose enough, so `nbcode` is free of implementation dependencies?

Reply:
Implementation dependency removed.